ui/src/stylesheets/admonitionblock.scss

```diff
@@ -39,14 +37,14 @@
       margin: 0;
       word-wrap: break-word;
       a {
-        word-break: break-all;
+        font-weight: var(--weight-bold);
```

Comment:
I feel that this should be consistent with the links in the body as well. My preference is for no bold either in the body or in the box links. What do you think?

Reply:
Good catch, agree with you